### PR TITLE
feat(examples): add two examples using auth plugins

### PR DESCRIPTION
**Pull request by Camji55**
Adds an example using [nexus-plugin-jwt-auth](https://github.com/Camji55/nexus-plugin-jwt-auth).

### CODE REVIEW

**Comment by jasonkuhrt** [comment]
Would be nice to see a convention form for examples pertaining to particular plugins. Thinking they can be called by their package name. But we can probably drop the `nexus-` prefix. So for example this one:

```
/plugin-jwt-auth
````

CC @Weakky

**Comment by Camji55** [comment]
@jasonkuhrt I think that naming suggestion makes sense, however I'm also working on an example that would use `jwt-auth` for the authentication, but using @lvauvillier's https://github.com/lvauvillier/nexus-plugin-shield for the permissions. In that case, where the example uses multiple plugins, how would that work in this convention?

**Comment by jasonkuhrt** [comment]
@Camji55 good question, thought a bit about it.

My current thinking is a convention of:

```
plugins-<name>-and-<name>[-and-<name>]*
```

The `s` (`plugin-` vs `plugins-`) will make grouping better. I think examples using multiple plugins are distinct from those using just one. The just-one cases can be seen as canonical for each respective plugin while the number of combos of plugins is/will be practically unbounded.

Directory name conventions are not/cannot be expected to solve all navigation use-cases. I think we will need a README that indexes the examples by different dimensions. For example:

- By domain: auth, perf, integration, ...
- By Level: simple, advanced
- By plugin (because a plugin may show up in multiple combo examples)

It should be pretty easy to generate that by scanning the package.json's of all the examples. Maybe we can ask example authors to use package tags to help us group them (e.g. the domain dimension) or markdown frontmatter...

Anyways, in conclusion for now, about the second example you mentioned, then the dir name for that would be:

```
plugins-jwt-auth-and-shield
```

**Comment by Camji55** [comment]
@jasonkuhrt Sounds like a plan!

**Comment by Camji55** [comment]
Folder names have been updated 👍

File: plugin-jwt-auth/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "auth",
+  "dependencies": {
+    "bcryptjs": "2.4.3",
+    "jsonwebtoken": "8.5.1",
+    "nexus": "0.21.0-next.14",
+    "nexus-plugin-jwt-auth": "1.0.0",
+    "nexus-plugin-prisma": "0.7.0"
```
Comment (jasonkuhrt) [inline plugin-jwt-auth/package.json:8]: To follow the convention this example should be called `plugins-jwt-auth-and-prisma`.

File: plugins-jwt-auth-and-shield/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "auth",
+  "dependencies": {
+    "bcryptjs": "2.4.3",
+    "jsonwebtoken": "8.5.1",
+    "nexus": "0.21.0-next.14",
+    "nexus-plugin-jwt-auth": "1.0.0",
+    "nexus-plugin-prisma": "0.7.0",
+    "nexus-plugin-shield": "^0.1.2"
```
Comment (jasonkuhrt) [inline plugins-jwt-auth-and-shield/package.json:9]: To follow the convention this example should be called `plugins-jwt-auth-and-shield-and-prisma`.

**Comment by Camji55** [comment]
Name updated 👍

**Pull request merged**